tests/test_threadpoolctl.py

```diff
@@ -302,6 +302,7 @@ def test_nested_prange_blas(nthreads_outer):
     assert original_info == _threadpool_info()
 
 
+@pytest.mark.filterwarnings("ignore::UserWarning")
```

Comment:
Could you please add a comment that explains both why we UserWarning to be raised in this test and why this is ok to ignore it here (insert of asserting that this warning is raised in 100% of the case for instance).